Makefile

```diff
@@ -0,0 +1,5 @@
+build:
+	sed -e "s/\$${UNISON_VERSION}/2.48.3/"   -e "s/\$${OCAML_VERSION}/4.02/" Dockerfile > Dockerfile-unison2.48.3-ocaml4.02
+	sed -e "s/\$${UNISON_VERSION}/2.48.3/"   -e "s/\$${OCAML_VERSION}/4.01/" Dockerfile > Dockerfile-unison2.48.3-ocaml4.01
+	sed -e "s/\$${UNISON_VERSION}/2.40.102/" -e "s/\$${OCAML_VERSION}/4.02/" Dockerfile > Dockerfile-unison2.40.102-ocaml4.02
+	sed -e "s/\$${UNISON_VERSION}/2.40.102/" -e "s/\$${OCAML_VERSION}/4.01/" Dockerfile > Dockerfile-unison2.40.102-ocaml4.01
```

Comment:
My shell skills are lacking and I'm struggling to get the for loops to work. I like the idea, but I'm going to stick with the super simple sed for the moment to give me some hope of maintaining it. Once we get this out I might try and wrangle this better.